eth_crowdfund_be/eth_crowdfund_api/src/app.py

```diff
@@ -3,6 +3,7 @@
 from .config import app_config
 from .models import db
 
+from .views.CampaignView import campaign_api as campaign_blueprint
```

Comment:
"A Blueprint is a way to organize a group of related views and other code." - internet